packages/auth/src/index.ts

```diff
@@ -104,7 +104,16 @@ export function initIocContainer(
 
   container.singleton('openApi', async (deps) => {
     const config = await deps.use('config')
-    return await createOpenAPI(config.authServerSpec)
+    const clientSpec = await createOpenAPI(config.authServerSpec)
```

Comment:
What if `get-op-schema.sh` also fetched the auth server spec, and we just used the local file (remove `config.authServerSpec`) as with the other specs?
That way we ensure all the OpenAPI specs in `auth` are using the same version of the schemas.